apps/browser/qml/pages/components/BookmarkEditDialog.qml

```diff
@@ -83,20 +83,34 @@ UserPromptDialog {
                 id: urlField
                 text: url
 
-                acceptableInput: text.length > 0
```

Comment:
Currently if there's an error (i.e. no http:// or https:// prefix) I can still accept the dialogue and create the app icon. I think it'd make sense to block accept in this case (`acceptableInput` doesn't seem to be tied to the validator by default for me).

Reply:
Unless the binding has been overridden TextField.acceptableInput == TextInput.acceptableInput which is internally determined from https://doc.qt.io/qt-5/qml-qtquick-textinput.html#acceptableInput-prop, and the errorHighlight is derived from acceptableInput so if you're seeing the error message then the value is being updated based on the validator.

It's possible you were previously able to accept with a empty URL or the acceptableInput wasn't used in that determination.

Reply:
The value TextField.acceptableInput is changing but Dialog.canAccept isn't beyond evaluating to true during construction.  I can't see any assignments to canAccept inside the qml file, so I guessing something external is breaking the binding.

Reply:
In practice what I'm finding is that the dialog can be accepted even with an error showing. See here (Save button isn't dimmed, selecting it still creates the app shortcut, but the shortcut doesn't work):
![acceptableInput](https://user-images.githubusercontent.com/1446122/122525143-99564c80-d021-11eb-90ce-3a5ff5f7436a.png)

Reply:
AddToAppGridDialog overwrote canAccept. Fixed.

Reply:
Thanks; working nicely now.